django/db/models/fields/__init__.py

```diff
@@ -2792,14 +2792,24 @@ def __init__(self, *args, **kwargs):
     def check(self, **kwargs):
         return [
             *super().check(**kwargs),
-            *self._check_primary_key(),
+            *self._check_primary_key(kwargs.get("databases", ["default"])),
```

Comment:
One of the tests I saw was passing `databases` kwarg into `check()` but in another test it was absent.  Is this a correct assumption here that `databases` may be passed in?